packages/create-sitecore-jss/src/templates/angular/scripts/install-pre-push-hook.js

```diff
@@ -0,0 +1,24 @@
+﻿const fs = require('fs');
+
+const installHooks = () => {
+  // data to be written to the file
+  const data = `#!/bin/sh
+#
+# pre-push hook runs our linter before we push our changes
+#
```

Comment:
Would be nice to mention --no-verify option to bypass like we do here: https://github.com/Sitecore/jss/blob/dev/scripts/install-hooks.js#L10-L11